### PR TITLE
[INLONG-6563][Manager] It should throw error info when deleting an inlong group failed

### LINKED ISSUE #6563: [Bug][Manager] No error message is displayed when the group fails to be deleted
### What happened

No error message is displayed when the group fails to be deleted

### What you expected to happen

An error message is displayed when the group fails to be deleted

### How to reproduce

Configure the group, create conditions that can cause group deletion failure, for example, deleting the cluster, and delete the group.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
- Fixes #6563 

### Motivation

Fix the problem of no error message was displayed when deleting a group failed

### Modifications

Get the execution result of deleting the group workflow, and throw an exception if the deletion fails

### CODE REVIEW

**Review by vernedeng — APPROVED** [review]
lgtm

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 1]
```diff
@@ -294,7 +294,7 @@ private void invokeDeleteProcess(String groupId, String operator) {
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.DELETE);
         WorkflowResult result = workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
         List<TaskResponse> tasks = result.getNewTasks();
-        if (TaskStatus.FAILED == tasks.get(tasks.size() - 1).getStatus()) {
+        if (TaskStatus.FAILED.equals(tasks.get(tasks.size() - 1).getStatus())) {
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:297]: why not to use `== ` for enum

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 2]
```diff
@@ -289,7 +292,13 @@ private void invokeDeleteProcess(String groupId, String operator) {
         InlongGroupInfo groupInfo = groupService.doDeleteCheck(groupId, operator);
         // start to delete group process
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.DELETE);
-        workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        WorkflowResult result = workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED.equals(tasks.get(tasks.size() - 1).getStatus())) {
+            String errMsg = String.format("failed to delete group process for groupId=%s", groupId);
+            LOGGER.error(errMsg);
```
Comment (luchunliang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:299]: It is better that
LOGGER.error("failed to delete group process for groupId={}", groupId);

when log level is fatal, errMsg object will not be created.

Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:299]: 1. We are all at least at the `info` level. No matter in the test or production environment, we use logs at the `info` level at least;
2. The following `WorkflowListenerException` also uses this error message, so it reuses the error message.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 3]
```diff
@@ -289,7 +292,13 @@ private void invokeDeleteProcess(String groupId, String operator) {
         InlongGroupInfo groupInfo = groupService.doDeleteCheck(groupId, operator);
         // start to delete group process
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.DELETE);
-        workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        WorkflowResult result = workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED == tasks.get(tasks.size() - 1).getStatus()) {
+            String errMsg = String.format("failed to delete group process for groupId=%s", groupId);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:298]:
```suggestion
            String errMsg = String.format("failed to delete inlong group for groupId=%s", groupId);
```

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Pull request merged**